test/storage/index.js

```diff
@@ -192,8 +192,4 @@ describe('Storage', function() {
       });
     });
   });
-
-  describe('makeReq_', function() {
```

Comment:
Blames me for what? It looks like this PR removes the tests for makeReq_, and the blame shows where I added them. Feel like I'm definitely missing out on something. Halp.